Sources/Growth/AppReviewManager.swift

```diff
@@ -14,11 +14,61 @@ import BraveVPN
 /// Singleton Manager handles App Review Criteria
 public class AppReviewManager: ObservableObject {
   
+  struct Constants {
+    // Legacy Review Constants
+    static let firstThreshold = 14
+    static let secondThreshold = 41
+    static let lastThreshold = 121
+    static let minimumDaysBetweenReviewRequest = 60
+
+    // Revised Review Constants
+    static let launchCountLimit = 5
+    static let bookmarksCountLimit = 5
+    static let playlistCountLimit = 5
+    static let dappConnectionPeriod = AppConstants.buildChannel.isPublic ? 7.days : 7.minutes
+    static let daysInUseMaxPeriod = AppConstants.buildChannel.isPublic ? 7.days : 7.minutes
+    static let daysInUseRequiredPeriod = 4
+    static let revisedMinimumDaysBetweenReviewRequest = 30
+  }
+  
+  /// A enumeration for which type of App Review Logic will be used
+  /// Helper for quick changes between different types of logic
+  public enum AppReviewLogicType: CaseIterable {
+    case legacy
+    case revised
+    case revisedCrossPlatform
```

Comment:
Can you add comments for each case explaining them? Hard to tell difference between `revised` and `revisedCrossPlatform`